client/state/selectors/get-jetpack-credentials-test-status.js

```diff
@@ -0,0 +1,7 @@
+import { get } from 'lodash';
+
+import 'calypso/state/jetpack/init';
+
+export default function getJetpackCredentialsTestStatus( state, siteId, role ) {
```

Comment:
Selectors are usually also good candidates for writing some unit tests, to ensure they work well in all instances (when we have the data and when we don't).

Reply:
Tests has been added for this selector :)